### PR TITLE
Fixed the iModel extent update test to not delete random iModels

### PR DESCRIPTION
**Pull request by austeja-bentley**
Couple of improvements for iModelHub client tests:
- Removed the custom project usage from setup
- Removed last iModel deletion from extent test

### CODE REVIEW

File: full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts [hunk 1]
```diff
@@ -258,8 +260,8 @@ describe("iModelHub iModelsHandler", () => {
     requestContext = new AuthorizedClientRequestContext(accessToken);
 
     (requestContext as any).activityId = "iModelHub iModelsHandler";
-    projectId = await utils.getProjectId(requestContext, "iModelJsTest");
```
Comment (calebmshafer) [inline full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts:261]: We were using a separate project due to the tests deleting random iModels in the other test project.  Does PR fix this issue?

Comment (austeja-bentley) [reply full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts:261]: Yes. The test that this PR fixes is "should update iModel extents" and the changes here remove the call to iModelClient.iModel.delete method which deletes the random (actually not random but the oldest) iModel.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**